code/extension/ShareThisSTE.php

```diff
@@ -91,50 +124,67 @@ public function ShowShareIcons()
         }
     }
 
-    public function ShareIcons()
+    /**
+     * Get the sharing icons
+     */
+    public function getShareIcons()
     {
         $bookmarks = $this->makeBookmarks('IncludeThisIcon');
         return $this->makeShareIcons($bookmarks);
     }
 
+    /**
+     * Grabbing front end dependencies for the expanded sharing list with some extra
+     * functionality
+     */
     public function ShareAllExpandedList()
     {
-        Requirements::javascript(FRAMEWORK_DIR . '/thirdparty/jquery/jquery.js');
+        Requirements::javascript('/resources/vendor/silverstripe/admin/thirdparty/jquery/jquery.min.js');
```

Comment:
The path '/resources/vendor' doesn't work - don't think we need the vendor bit. 
It looks like the preferred way of handling paths in SS4 is something like:
`Requirements::javascript('silverstripe/admin: thirdparty/jquery/jquery.min.js');`

Reply:
Done!